README.md

```diff
@@ -22,22 +22,22 @@ npm i doshii-sdk-2.0.0.tgz
 
 The `Doshii` constructor accepts credentials and is thereafter used for all the API calls.
 
-All the API s are properties of the main `Doshii` class, which can be invoked as required. The API s include `location`, `checkin`, `device`, `loyalty`, `menu`, `order`, `table`, `transaction`, `webhook` and `booking`.
+All the API's are properties of the main `Doshii` class, which can be invoked as required. The API s include `location`, `checkin`, `device`, `loyalty`, `menu`, `order`, `table`, `transaction`, `webhook` and `booking`.
```

Comment:
Not a showstopper but ironically, I think it was actually correct grammatically without the apostrophe? Maybe should we remove the space instead? APIs?

Reply:
Ah, yes, good point - thanks. 🙇